src/DependencyInjection/ConnecthollandUserExtension.php

```diff
@@ -32,13 +32,34 @@ public function load(array $configs, ContainerBuilder $container)
             new FileLocator(__DIR__.'/../Resources/config')
         );
 
+        if (isset($container->getParameter('kernel.bundles')['LexikJWTAuthenticationBundle'])) {
+            $loader->load('lexik_jwt_authentication.yaml');
+        }
+
+
         $loader->load('services.yaml');
     }
 
     public function prepend(ContainerBuilder $container)
     {
         $config = $this->getResourceOwnersConfiguration($container);
         $container->prependExtensionConfig('hwi_oauth', $config);
+
+        if ($container->hasExtension('lexik_jwt_authentication')) {
+            $config = $this->getJwtConfiguration($container);
+            $container->prependExtensionConfig('lexik_jwt_authentication', $config);
+        }
+    }
+
+    private function getJwtConfiguration(ContainerBuilder $container): array
+    {
+        $config = [
+            'secret_key'  => $container->resolveEnvPlaceholders($container->getParameter('env(JWT_SECRET_KEY)'), true),
+            'public_key'  => $container->resolveEnvPlaceholders($container->getParameter('env(JWT_PUBLIC_KEY)'), true),
+            'pass_phrase' => $container->resolveEnvPlaceholders($container->getParameter('env(JWT_PASSPHRASE)'), true),
```

Comment:
Doesn't this error on cache warming? As JWT_PASSPHRASE is not available as environment variable ->  https://github.com/ConnectHolland/user-bundle/pull/44/files#diff-9dcf13693e42690b6b8596e9972cd468R3

Reply:
Added an empty default for now.